posts/2022-02-08-Rust-Survey-2021.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Rust Survey 2021 Results"
+author: The Rust Survey Team
+---
+
+# Blog Post
+
+Greetings Rustaceans!
+
+Another year has passed, and with it comes another annual Rust survey analysis! The survey was conducted in December 2021.
+
+We’d like to thank everyone who participated in this year’s survey with a special shout-out to those who helped translate the survey from English into other languages.
+
+Without further ado, let’s dive into the analysis!
+
+## A Global Community
+
+The Rust community continues to grow, with this survey having the largest number of complete survey responses (9354 respondents), exceeding last year's total by roughly 1500 responses.
+
+90% of respondents said that they use Rust for any purpose including learning (an all-time high!) with 5% stating they have in the past and 4% stating they have yet to use Rust.
+
+The survey was offered in 10 languages with 78% filling out the survey in English followed by Chinese (6%), German (4%), and French (3%). Despite English being the language most respondents completed the survey in, respondents hailed from all around the world. The United States was the country with the largest representation at 24% followed by Germany (12%), China (7%), and the U.K. (6%). In total 113 different countries and territories were represented through this survey!
+
+English, however, is not the language of choice for all Rustaceans with nearly 7% preferring not to use English as a language for technical communication. An additional 23% of respondents prefer another language in addition to English. The most common prefered languages (besides English) follow where Rustaceans live: Chinese, German, and French being the top 3. Japanese, Chinese, and Russian speakers were the most likely to prefer not to use English for technical conversation.
```

Comment:
"follow where Rustaceans live: Chinese, German, and French being the top 3" 

This seems oddly ordered -- the previous paragraph suggests Germany has more respondents than China? Maybe we meant to say "the choice of language for filling out the survey"?

Reply:
I added the term "roughly" since it's true that it doesn't follow the order exactly. Is that enough?

Reply:
Yeah, seems good.